apps/game-service/src/options.ts

```diff
@@ -46,6 +45,12 @@ export const gameServiceOptionsSchema = type({
     interval: msSchema,
   },
   databaseConnectionString: "string",
+  /**
+   * The URL to the Electric sync service (optional).
+   * When provided, enables real-time database sync via Electric.
+   * When not provided, falls back to polling-based sync.
+   */
+  "electricUrl?": "string",
```

Comment:
it should not be optional. also you have no added the required docker infrastructure for this.